### ISSUE TITLE
Have you ever repeated the PSNR results from paper?

### ISSUE THREAD

**Issue by sjtu-nobody** [LoSealL/VideoSuperResolution#80]
Hi, I am a graduate student from SJTU and an newcomer to the SR field. Recently I try to evaluate a 2× SR network structure with SRCNN and ESPCN as benchmark.  But SRCNN and ESPCN I trained using 91-images set only gets 31.4 and 31.5 PSNR on Set14, which is much lower than the results in the paper under same setting. Your repository is the only one I find contains detailed retrained PSNR results and it seems you are quite experienced in this area, so I wonder whether you could provide some advice on how to get similar results in the paper?

**Comment by LoSealL** [comment]
Hi @sjtu-nobody 
If you are familiar with ML but new to SR, there are a few advises:
1. Details are the most important thing. What's your optimizer v.s. paper's optimizer? Did you decay your learning rate and how? How do you expand your dataset?
2. The default settings in this repository are just tuned for my test-cases and only for x4 scale factor, therefore you have to tune hyper-parameters for other cases.
3. 91-image is just a toy dataset, too small for SR.
